### PR TITLE
 BaSP-MR-22: finish update and delete

### PR DESCRIPTION
**Pull request by GinoLucci1**
Finish this part of the weekly problem.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "dotenv": "^16.0.3",
     "express": "^4.18.0",
     "joi": "^17.9.2",
+    "mongodb": "^5.5.0",
```
Comment (maurojjzz) [inline package.json:33]: you should not modify package.json

File: package-lock.json
Comment (maurojjzz) [inline package-lock.json:1]: i dont know what you change in here but you shouldn't push it or change it

**Review by maurojjzz — CHANGES_REQUESTED** [review]

**Comment by Bigzufi** [comment]
For my part, it looks good, once you have made the changes suggested by Mauro

**Review by maurojjzz — APPROVED** [review]
seems fine

**Review by Bigzufi — APPROVED** [review]
Looks good.

**Review by BareaMartin — APPROVED** [review]

**Pull request merged**